SensorsAnalyticsSDK/src/main/java/com/sensorsdata/analytics/javasdk/SensorsAnalytics.java

```diff
@@ -1402,7 +1415,11 @@ private static ObjectMapper getJsonObjectMapper() {
     return jsonObjectMapper;
   }
 
-  private static final String SDK_VERSION = "3.1.17";
+  private static CloseableHttpClient getCloseableHttpClient() {
+    return HttpClients.custom().setUserAgent("SensorsAnalytics Java SDK " + SDK_VERSION).build();
+  }
+
+  public static final String SDK_VERSION = "3.1.17";
```

Comment:
We make the `SDK_VERSION` public so that users can recreate the default User-Agent setting for the HTTP client themselves.